projects/helper/coreAssets.json

```diff
@@ -2241,7 +2241,13 @@
   "plume": {
     "USDC_e": "0x3938A812c54304fEffD266C7E2E70B48F9475aD6",
     "USDT": "0xA849026cDA282eeeBC3C39Afcbe87a69424F16B4",
-    "WETH": "0x626613B473F7eF65747967017C11225436EFaEd7"
+    "WETH": "0x626613B473F7eF65747967017C11225436EFaEd7",
+    "pUSD": "0xdddD73F5Df1F0DC31373357beAC77545dC5A6f3F",
+    "pETH": "0xD630fb6A07c9c723cf709d2DaA9B63325d0E0B73",
+    "nRWA": "0x81537d879ACc8a290a1846635a0cAA908f8ca3a6",
+    "nTBILL": "0xE72Fe64840F4EF80E3Ec73a1c749491b5c938CB9",
+    "nYIELD": "0x892DFf5257B39f7afB7803dd7C81E8ECDB6af3E8",
+    "nELIXIR": "0x9fbC367B9Bb966a2A537989817A088AFCaFFDC4c"
```

Comment:
Please remove this change. It has no effect on your adapter. Who mints nRWA, nTBILL, nYIELD, nELIXIR?